### PR TITLE
Add support for metricRelabelings to the helm chart - DI-6087

### PR DESCRIPTION
**Pull request by ferrous26**
Hello, I wanted to add some metric relabelings to adjust some of the standard labels. Partly to get an HA setup that doesn't duplicate time series in prometheus due to pod/instance labels.

https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md

### CODE REVIEW

File: helm/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,7 @@ apiVersion: v2
 name: sql-exporter
 description: Database agnostic SQL exporter for Prometheus
 type: application
-version: 0.4.0
-appVersion: 0.14.0
```
[the remark this answers is not in the document]
Comment (ferrous26) [reply helm/Chart.yaml:6]: Oops, I didn't realize that I had removed that line.

**Comment by ferrous26** [comment]
> Hey @ferrous26, looks good to me. 👍 Do you mind to set the chart version to `0.4.1` instead of `0.5.0`? So far I preserve minor number for new versions or breaking changes. This change seems rather minor. 🙂
> 
> Also, please update the chart docs by following this section - [link](https://github.com/burningalchemist/sql_exporter/tree/master/helm#dev-notes) (`make gen_docs`).

Done.

File: helm/Chart.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (burningalchemist) [reply helm/Chart.yaml:6]: Yeah, no worries. 😉

**Review by burningalchemist — APPROVED** [review]
LGTM! 🚀

**Pull request merged**